src/main/kotlin/io/snyk/plugin/services/SnykAnalyticsService.kt

```diff
@@ -145,4 +146,8 @@ class SnykAnalyticsService : Disposable {
             log.warn("Failed to execute '$message' analytic event. ${t.message}", t)
         }
     }
+
+    fun logAuthenticateButtonIsClicked(event: AuthenticateButtonIsClicked) {
+        TODO("Not yet implemented")
+    }
```

Comment:
sems like we need to implement that

Reply:
Yep, definitely forgot that implementation 😂